src/ngrx/5-dispatch-actions/dispatch-actions.md

```diff
@@ -5,4 +5,46 @@
 
 @body
 
-TODO
\ No newline at end of file
+> **Quick Start**: You can checkout [this branch](https://github.com/bitovi/angular-ngrx-chat/tree/create-actions) to get your codebase ready to work on this part.
+
+## Overview
+
+1. Update `LoginComponent` to dispatch `LoginActions.login`
+
+2. Update `DashboardComponent` to dispatch `LoginActions.logout`
```

Comment:
not something to change, but it is interesting how createActionGroup will change this to LoginActios.login and DashboardActions.logout maybe 🤔

Reply:
yea, I'd be curious how this will affect the norm moving forward

Reply:
I'm going to keep this comment unresolved just in case it opens more discussion, then will close once all other comments are resolved and all approvals are done